src/routes/delete.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { posts } from '../posts';
+
+const router = Router();
+
+router.delete('/api/posts/:id', (req, res) => {
+  // Check if the post already exists
+  const post = posts.find((post) => post.id === +req.params.id);
```

Comment:
What is this + for?

Reply:
It is a short numeric conversion using the plus + operator, which is equivalent to Number(req.params.id) and parseInt(req.params.id).